lit/docs/guides/git.lit

```diff
@@ -139,3 +139,230 @@
+    In this case, let's assume our feature branches match the regular expression
+    \code{feature/.*}.
+
+    Below is the full pipeline config for this tracker pipeline:
+
+    \link{\code{examples/pipelines/multi-branch/tracker.yml}}{https://github.com/aoldershaw/examples/blob/master/pipelines/multi-branch/tracker.yml}
+    \codeblock{yaml}{{{
+      resource_types:
+      - name: git-branches
+        type: registry-image
+        source:
+          repository: aoldershaw/git-branches-resource
+
+      resources:
+      - name: feature-branches
+        type: git-branches
+        source:
+          uri: https://github.com/concourse/examples
+          branch_regex: 'feature/(?P<feature>.*)'
+
+      - name: examples
+        type: git
+        source:
+          uri: https://github.com/concourse/examples
+
+      jobs:
+      - name: set-feature-pipelines
+        plan:
+        - in_parallel:
+          - get: feature-branches
+            trigger: true
+          - get: examples
+        - load_var: branches
+          file: feature-branches/branches.json
+        - across:
+          - var: branch
+            values: ((.:branches))
+          set_pipeline: dev
+          file: examples/pipelines/multi-branch/template.yml
+          instance_vars: {feature: ((.:branch.groups.feature))}
+          vars: {branch: ((.:branch.name))}
+    }}}
+
+    We set each pipeline as an \reference{instanced-pipelines}{instanced
+    pipeline} - this will result in Concourse grouping all of the related
+    \code{dev} pipelines in the UI.
+  }
+
+  \section{
+    \title{Cleaning Up Old Workspaces}{multi-branch-cleaning-up}
+
+    With the setup described in \reference{multi-branch-tracking-branches},
+    Concourse will automatically archive any pipelines for branches that get
+    removed. However, Concourse doesn't know that it should destroy Terraform
+    workspaces when a branch is removed. To accomplish this, we can yet again
+    make use of the \link{Terraform resource}{https://github.com/ljfranklin/terraform-resource}
+    to destroy these workspaces. We'll add another job to the tracker pipeline
+    that figures out which workspaces don't belong to an active branch and
+    destroy them.
+
+    \link{\code{examples/pipelines/multi-branch/tracker.yml}}{https://github.com/aoldershaw/examples/blob/master/pipelines/multi-branch/tracker.yml}
+    \codeblock{yaml}{{{
+      resource_types:
+      - name: git-branches
+        ...
+
+      - name: terraform
+        type: registry-image
+        source:
+          repository: ljfranklin/terraform-resource
+
+      resources:
+      - name: feature-branches
+        ...
+
+      - name: examples
+        ...
+
+      - name: staging-env
+        type: terraform
+        source:
+          backend_type: gcs
+          backend_config: &terraform_backend_config
+            bucket: concourse-examples
+            prefix: multi-branch/terraform
+            credentials: ((concourse_artifacts_json_key))
```

Comment:
nit
```suggestion
            credentials: ((artifacts_json_key))
```

Reply:
Changed to `((gcp_service_account_key))` - probably a bit more obvious